osmosis-1/osmosis.zone_assets.json

```diff
@@ -1566,6 +1566,13 @@
       "path": "transfer/channel-1411/umpwr",
       "osmosis_main": false,
       "osmosis_frontier": true
+    },
+    {
+      "chain_name": "xpla",
+      "base_denom": "axpla",
+      "path": "transfer/channel-1411/umpwr",
```

Comment:
this path is copied from the MPWR asset. I couldn't correct this because there is no osmosis-xpla ibc connection registered in the Chain Registry yet